src/tests/functional/plugin/shared/CMakeLists.txt

```diff
@@ -69,6 +69,7 @@ ov_add_target(
                 sharedTestClasses
             PRIVATE
                 ${LINK_LIBRARIES_PRIVATE}
+                OpenCL
```

Comment:
This dependency should be only for GPU build not for all test.
Add skip tests in GPU only